test/GithubApi.Repositories.test.js

```diff
@@ -0,0 +1,66 @@
+const axios = require('axios');
+const { expect } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+const md5 = require('md5');
+
+const path = 'https://api.github.com/users/aperdomob';
+const path2 = 'https://api.github.com/repos/aperdomob';
+
+const query1 = {
+  name: 'Alejandro Perdomo',
+  company: 'Perficient Latam',
+  location: 'Colombia'
+};
+
+const query2 = {
+  full_name: 'aperdomob/jasmine-json-report',
+  private: false,
+  description: 'A Simple Jasmine JSON Report'
+};
+
+const query3 = {
+  name: 'README.md',
+  path: 'README.md',
+  sha: '360eee6c223cee31e2a59632a2bb9e710a52cdc0'
+};
+
+let result = null;
+
+describe('Metodos GET', () => {
+  it('Verficacion del usuario', async () => {
+    const response = await axios.get(`${path}`);
+
+    expect(response.status).to.equal(StatusCodes.OK);
+    expect(response.data.name).to.equal(query1.name);
+    expect(response.data.company).to.equal(query1.company);
+    expect(response.data.location).to.equal(query1.location);
+  });
+
+  it('Verficacion del contenido del repositorio', async () => {
+    const response = await axios.get(`${path}/repos`);
+    const repos = response.data;
+    result = repos.find(({ name }) => name === 'jasmine-json-report');
```

Comment:
Use a more descrptive name for the variable. Try to think, what is result? Is the result of what?. Try with a name like `jasmineReportRepository` or something similar